helm/pyroscope/values.yaml

```diff
@@ -1,7 +1,38 @@
+pyroscopeagent:
```

Comment:
Maybe `pyroscopeAgent` (camelCase) would be easier to read?
BTW, the schema already uses camelCase.

Reply:
That's what I was using a few minutes ago but upon deployment I was running into errors for sa's names that are not allowing camelCase for their names :/